src/schemas/json/tsconfig.json

```
@@ -292,13 +301,13 @@
                     "ES2020",
                     "ESNext",
                     "None",
-                    "es2022",
-                    "node12",
-                    "nodenext"
+                    "ES2022",
+                    "Node12",
+                    "NodeNext"
                   ]
                 },
                 {
-                  "pattern": "^([Cc][Oo][Mm][Mm][Oo][Nn][Jj][Ss]|[AaUu][Mm][Dd]|[Ss][Yy][Ss][Tt][Ee][Mm]|[Ee][Ss]([356]|201[567]|2020|[Nn][Ee][Xx][Tt])|[Nn][Oo][Nn][Ee])$"
+                  "pattern": "^([Cc][Oo][Mm][Mm][Oo][Nn][Jj][Ss]|[AaUu][Mm][Dd]|[Ss][Yy][Ss][Tt][Ee][Mm]|[Ee][Ss]([356]|201[567]|202[0|2]|[Nn][Ee][Xx][Tt])|[Nn][Oo][Nn][Ee])$"
```

Consider using the same patterns as for `"target"`. Also it should be `[02]` instead of `[0|2]`.
```suggestion
                  "pattern": "^([Cc][Oo][Mm][Mm][Oo][Nn][Jj][Ss]|[AaUu][Mm][Dd]|[Ss][Yy][Ss][Tt][Ee][Mm]|[Ee][Ss]([356]|20(1[567]|2[02])|[Nn][Ee][Xx][Tt])|[Nn][Oo][Nn][Ee])$"
```

---

done